src/ur/workday.ur

```diff
@@ -0,0 +1,378 @@
+     HiddenFromManager : option bool, (* True if the feedback event is private between the feedback provider and the worker. Private feedback isn't shared with managers. *)
+     Comment : option string, (* actual feedback *)
+     FromWorker : option descriptor, (* Identity of the responder. Only provided if the request asked to know the identity of the responder. *)
+     OverallStatus : option string,
+     GivenDate : option time,
+     ShowFeedbackProviderName: option bool,
+     (* unused parameters: *)
+     (*
+     Badge : descriptor,
+     Href : string
+     AlsoAbout : list descriptor, (* other workers the feedback is about *)
+     RelatedFeedbackEvents : list relatedFeedback,
+     RelatesTo : descriptor,
+      *)
+}
+val json_anytimeFeedback : json anytimeFeedback =
+    json_record_withOptional
+        {Id = "id"}
+        {Preview = "descriptor",
+         ToWorker = "toWorker",
+         WorkersToNotify = "workersToNotify",
+         BusinessProcessParameters = "businessProcessParameters",
+         HiddenFromWorker = "hiddenFromWorker",
+         HiddenFromManager = "hiddenFromManager",
+         Comment = "comment",
+         FromWorker = "fromWorker",
+         OverallStatus = "overallStatus",
+         GivenDate = "feedbackGivenDate",
+         ShowFeedbackProviderName = "showFeedbackProviderName"
+                 (*
+                 RelatesTo = "relatesTo",
+                 Badge = "badge",
+                 AlsoAbout = "feedbackAlsoAbout",
+                 RelatedFeedbackEvents = "relatedFeedbackEvents",
+                  *)
+                }
+
+datatype feedbackStatus = Complete | Requested
+(*
+val show_feedbackStatus : show feedbackStatus =
+ fn s => case s of
+             Complete => "Complete"
+           | Requested => "Requested"
+val read_feedbackStatus : read feedbackStatus
+val eq_feedbackStatus : eq feedbackStatus
+*)
+
+con feedback = [
+      Id = wid,
+      Created = time,
+      RequestedDate = option time,
+      About = wid, (* who the feedback is about *)
+      Provider = wid, (* who writes the feedback *)
+      RequestedBy = option wid,
+      Body = string, (* response *)
+      Guidance = string,
+      Status = string, (* TODO: use feedbackStatus *)
+      SendMail = bool,
+      LastUpdated = time
+]
+
+con feedbackWithoutId = [
+      Created = time,
+      RequestedDate = option time,
+      About = wid, (* who the feedback is about *)
+      Provider = wid, (* who writes the feedback *)
+      RequestedBy = option wid,
+      Body = string, (* response *)
+      Guidance = string,
+      Status = string, (* TODO: use feedbackStatus *)
+      SendMail = bool,
+      LastUpdated = time
+]
+
+(* ************************************************************************** *)
+
+functor Make(M : AUTH) = struct
+    open M
+
+    (*
```

Comment:
This comment doesn't belong here.  It seems to describe one particular app, not a general API for a SaaS service.

Reply:
Good note, removed